deps/GraphBLAS/Source/GxB_Matrix_tuple_iter.c

```diff
@@ -171,6 +175,8 @@ static bool _find_row_index_in_Ah
 	GB_RETURN_IF_NULL(A) ;
 	GB_RETURN_IF_NULL(result) ;
 
+	if(A->nvec == 0) return false;
```

Comment:
Here I'm not sure this optimising much since GB_BINARY_SEARCH checks if while (pleft < pright) and in the case A->nvec == 0 it will not get in to the while loop and return immediately, so if the more prevalent case is that the matrix isn't empty it just adds another branching.
Need to consider removing this line here.